src/main/resources/hudson/tasks/Mailer/global_it.properties

```diff
@@ -0,0 +1,36 @@
+# The MIT License
+#
+# Copyright � 2020 Alessandro Menti
```

Comment:
Just because the character might not be correctly rendered on different encoding

```suggestion
# Copyright (c) 2020 Alessandro Menti
```